docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -273,6 +273,11 @@ prob of adding 2nd drug if not at target = 0.06 (0.02);
 prob of switching drugs if not at target = 0.03 (0.02); 
 prob of no change (BAU only) = 0.84 (0.02)
 
+Note: patients in BAU will "ramp-up" their treatment if they are not "at target", which is defined as a 50% or more reduction in their untreated LDL-C.  Reduction in LDL-C due to treatment is determined based on the particular treatment given to the simulant, whether they are adherent and intervention efficacy data in Table 9.  Specifically:  LDL-C reduction = is_adherent(0 or 1)*reduction_in_ldlc for their particular drug/dose Rx.
+
+Note:  Ramp-up should be monotonic in terms of average efficacy:  lifestyle_intervention < fibrates < ezetimibe < LP LD statin < LP HD statin < HP LD statin < HP HP statin < 2-drug combination (either as 2-pill or FDC, depending on scenario). The probabilities in Table 5 above determin if dose will increase (from LD to HD) or if potency will increase (from LP to HP) or if a 2nd drug will be added (note BAU Table 5 includes "prob of no change", while intervention scenario Table 5 does not - so there are 2 Table 5s).  These are mutually exclusive events - only one can change at any given time doctor visit.  Thus, it is ok for a patient to transition from LP LD statin to HP LD statin in one step - even though they jumped over LP HD statin - since only potency changed in this example.   
```

Comment:
is table 5 your transition probability table that has rows for 15 treatment profiles * 15 treatment profiles * all locations?

Reply:
no.  it has 3 rows for intervention scenarios 1 and 2: prob(add 2nd drug); prob(change drug); prob(increase dose)
it 4 rows for BAU: prob(add 2nd drug); prob(change drug); prob(increase dose) and prob(change nothing)